CONTRIBUTING.md

```diff
@@ -62,16 +62,22 @@ Pull requests (PRs)
 - During a squash/fixup, use `exec make -C build unittest` between each
   pick/edit/reword.
 
-### Stages: WIP, RFC, RDY
+### Stages: Draft and Ready for review
 
-Pull requests have three stages: `[WIP]` (Work In Progress), `[RFC]` (Request
-For Comment) and `[RDY]` (Ready).
+Pull requests have two stages: Draft and Ready for review.
 
-1. `[RFC]` is assumed by default, **do not** put "RFC" in the PR title (it adds
-   noise to merge commit messages).
-2. Add `[WIP]` to the PR title if you are _not_ requesting feedback and the work
-   is still in flux.
-3. Add `[RDY]` to the PR title if you are _done_ and only waiting on merge.
+1. [Create a Draft PR][pr-draft] while you are _not_ requesting feedback as
+  you are still working on the PR.
+2. [Change your PR to ready][pr-ready] when the PR is ready for review.
+
+You can convert the state of your PR back to Draft (or Ready for review) at any
+time. You can also skip the Draft stage if your PR is ready for review from the
+beginning.
+
+Do __not__ add any labels like `[RFC]` or `[WIP]` in the title to indicate the
+state of your PR: this just adds noise. Non-Draft PRs are assumed to be open
+for comments by default; if you want feedback from specific people, `@`-ping
+them in a comment.
```

Comment:
Maybe add that you can ping teams directly too ? Like `@neovim/treesitter` and `@neovim/lsp` ?

Reply:
Good point, but as long as we don't have more teams, how useful will that be? I worry that everyone will just ping `@neovim/neovim` then?

Reply:
Maybe we could just have something like:
```
Remember that you can also ping teams by ping  `@neovim/...`
```

Reply:
This probably isn't worth the extra text in docs, which should be very selective about how much patience they expect from readers. If we have a team for treesitter, presumably they are looking for issues/PRs labeled treesitter.